backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/service/VirtualQueueService.java

```diff
@@ -40,4 +43,9 @@ public CreateShopResponse createShop(CreateShopRequest createShopRequest) {
 	public GetTokensResponse getTokens(UUID shopId) {
 		return new GetTokensResponse(shopId, virtualQueueRepository.getTokens(shopId));
 	}
+
+	public GetAllShopsResponse getAllShops() {
+		GetAllShopsResponse getAllShopsResponse = new GetAllShopsResponse();
+		return getAllShopsResponse.setShops(virtualQueueRepository.getAllShops());
```

Comment:
Club these 2 as well. 
return  new GetAllShopsResponse().setShops(virtualQueueRepository.getAllShops());
Similarly in the other PR.

Reply:
Ok Done